src/App.vue

```diff
@@ -674,10 +674,11 @@ export default {
     },
 
     storeNrData (nr: NameRequestIF, ia: any): void {
-      // check if NR is valid
-      if (!this.isNrValid(nr)) {
+      // verify that NR is valid
+      const error = this.isNrInvalid(nr)
+      if (error) {
         this.nameRequestInvalidDialog = true
-        throw new Error('Invalid NR data')
+        throw new Error(error)
```

Comment:
Won't this just log "true" as an error? I'm not sure what Error() does, but if its old argument was a string containing info on the error, would logging "true" be beneficial?